bodhi/client/bindings.py

```diff
@@ -400,12 +396,22 @@ def override_str(override):
         if isinstance(override, six.string_types):
             return override
 
-        # TODO -- make this fancy.
-        return "{submitter}'s {build} override (expires {expiry})".format(
-            submitter=override['submitter']['name'],
-            build=override['build']['nvr'],
-            expiry=override['expiration_date'],
-        )
+        if minimal:
+            return "{submitter}'s {build} override (expires {expiry})".format(
+                submitter=override['submitter']['name'],
+                build=override['build']['nvr'],
+                expiry=override['expiration_date'],
+            )
+
+        val = "%s\n%s\n%s\n" % ('=' * 60, '\n'.join(
+            textwrap.wrap(override['build']['nvr'].replace(',', ', '), width=60,
+                          initial_indent=' ' * 5, subsequent_indent=' ' * 5)), '=' * 60)
+        val += "  Submitter: {}\n".format(override['submitter']['name'])
+        val += "  Expiration Date: {}\n".format(override['expiration_date'])
+        val += "  Notes: {}\n".format(override['notes'])
+        val += "  Expired: {}".format(override['expired_date'] is not None)
+
+        return val
```

Comment:
This change seems fine in general, but it's also separate of adding editable buildroot overrides. Please break it into a second commit so each commit is atomic and specific in scope. You can include that commit in this same PR if you want to.